app/application.go

```diff
@@ -24,6 +28,32 @@ func NewApplication(ctx context.Context, config *config.Config) Application {
 	return &application{ctx: ctx, config: config}
 }
 
+func (a *application) InitJaegerTracer() func() {
```

Comment:
Putting these functions in a **separate file** is better to prevent cluttering the app file.